src/main/java/org/broadinstitute/hellbender/tools/copynumber/PreprocessIntervals.java

```diff
@@ -184,30 +233,109 @@ private static IntervalList padIntervals(final List<SimpleInterval> inputInterva
         return paddedIntervalList;
     }
 
-    private static IntervalList generateBins(final IntervalList preparedIntervalList, final int binLength, final SAMSequenceDictionary sequenceDictionary) {
+    private static Stream<Interval> generateBinStream(final IntervalList preparedIntervalList, final int binLength, final boolean grid,
```

Comment:
Signatures of methods in CNV tools typically have one argument per line with vertical alignment.